.github/workflows/main.yml

```diff
@@ -1,10 +1,13 @@
 name: main
 
 on:
-  push:
```

Comment:
Is removing this event a mistake?

Reply:
I'm happy to revert now we have the de-dupe, which I copied from your PR

Reply:
Without it, we were getting double-checks. Maybe it too should be restricted to `main` branch?

Reply:
See suggested change https://github.com/pact-foundation/composer-downloads-plugin/pull/7/files#r1527540973

Reply:
I am also feeling annoy with Github Actions's double-checks. It's a wait of Github's resource. But I don't have a good solution for it. Since it's not a mistake, please keep this change

Reply:
According to https://docs.github.com/en/actions/examples/using-scripts-to-test-your-code-on-a-runner#example-workflow, maybe this is a good combination:

```yaml
on:
  push:
    branches: [ main ]
  pull_request:
```

I'm not so sure